### DISCUSSION TITLE
Confusion with Understanding how to Answer question 1.17 on session 1 - Worksheet 01A

### DISCUSSION THREAD

**Discussion by CourtA918** [LiNk-NY/IntroR#34] [Q&A] [answered]
Hi all! For some reason, as I was answering each session, I found myself very lost by the instructions given on 1.17. Would anyone happen to be able to simplify what it is asking me to do? I keep receiving an error for each answer input.

**Comment by LiNk-NY** [comment]
Don't forget to tag @ralythompson24

**Reply by ralythompson24** [reply]
Thanks @LiNk-NY - I will remind them to do this. I think this student asked the same question in the main discussion board so I'll close this one.

**Comment by ralythompson24** [comment, marked as answer]
See answer posted in https://github.com/CUNYSPHCode/IntroR/discussions/33?sort=new#discussioncomment-9596103
